kitsune/wiki/tests/test_views.py

```diff
@@ -369,6 +369,16 @@ def test_new_document_slugs(self, get_current):
                                                args=[data['slug']],
                                                locale='en-US'))
 
+    def test_translate_with_parent_locale_slug(self):
+        doc = document(locale=settings.WIKI_DEFAULT_LANGUAGE, save=True)
+        trans_doc = document(parent=doc, locale='bn-BD', slug='bn-BD_slug', save=True)
+        url = reverse('wiki.edit_document', args=[doc.slug], locale=trans_doc.locale)
+        response = self.client.get(url)
+        assert response.status_code == 200
```

Comment:
Don't use bare assert statements, use the `eq_` helpfer from nose. It produces easier to debug tests.